Library/Homebrew/extend/os/mac/requirements/osxfuse_requirement.rb

```diff
@@ -0,0 +1,34 @@
+require "requirement"
```

Comment:
This file should have basically no substantive changes from the original, it's just been moved into a Mac-specific place.